rls-analysis/src/lowering.rs

```diff
@@ -242,10 +244,30 @@ impl<'a> CrateReader<'a> {
                     ve.insert(Ref::Id(def_id));
                 }
             }
-            analysis.ref_spans.entry(def_id).or_insert_with(|| vec![]).push(span);
+            analysis.ref_spans.entry(def_id).or_insert_with(Vec::new).push(span.clone());
+
+            #[cfg(feature = "idents")]
+            {
+                Self::record_ident(analysis, &span, def_id, IdentKind::Ref);
```

Comment:
We can compute this gated on this feature but maybe it might make sense to make it dynamically configurable in the loader or (even better?) make it lazy-computable when `Analysis::idents` is invoked?

Reply:
It is hard to do it lazily, we don't have the data available in an ordered or addresable format, so looking up all idents in a span is O(n) (n = number of idents in a project). I think we should wait until it is necessary to make it dynamically configurable. It might be a feature which ends up not being used much.